Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,58 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            int len = exactPostTime.Length;
+            string[] arr = new string[len];
+            for(int i=0;i<len-1;i++){
+                if(exactPostTime[i]==exactPostTime[i+1] && showPostTime[i]!=showPostTime[i+1]){
+                    return "impossible";
+                }
+            }
+
+           
+            for(int i=0;i<len;i++){
+
+                 TimeSpan timing = TimeSpan.Parse(exactPostTime[i]);

```

Comment:
TimeSpan to var.
'timing' to 'postTime' or something of your choice... Variable name should say what value it's gonna hold